docs: redesign docs landing and integrations overview (#16807)
ollama/ollama: docs/integrations/index.mdx

title: Overview
---

Use Ollama from coding agents, personal assistants, and editors.

> Open an integration page for setup steps, or run `ollama launch` to configure a supported tool from the terminal.

Reviewer: ```suggestion
```

Reviewer: Just make a comment saying run `ollama launch` to see the latest integrations to run

title: Overview
---

Use Ollama from coding agents, personal assistants, and editors.

Open an integration page for setup steps, or run `ollama launch` to configure supported tools from the terminal.

## Code in the terminal

> Use these when you want an agent that can read, edit, and run code from your shell.

Reviewer: ```suggestion
```

<CardGroup cols={2}>
  <Card title="Claude Code" icon="/images/launch-icons/claude-code.svg" href="/integrations/claude-code">
    Terminal coding agent with tools, vision, web search, and long context.
  </Card>

  <Card title="OpenCode" icon="/images/launch-icons/opencode.svg" href="/integrations/opencode">
    Open-source coding agent that edits, runs, and iterates on code.
  </Card>
</CardGroup>

## Connect an assistant

Use these when you want an assistant with memory, skills, or messaging app access.

<CardGroup cols={2}>
  <Card title="OpenClaw" icon="/images/launch-icons/openclaw.svg" href="/integrations/openclaw">
    Personal assistant for messaging apps and everyday tasks.
  </Card>

  <Card title="Hermes Agent" icon="/images/launch-icons/hermes-agent.svg" href="/integrations/hermes">

>     Agent with memory, skills, and optional messaging gateways.

Reviewer: ```suggestion
    Open source  agent with self-improving skills, memory, and messaging.
```

  </Card>
</CardGroup>

## Work in your editor

Use Ollama models inside your editor.

<CardGroup cols={2}>
  <Card title="VS Code" icon="/images/launch-icons/vscode.svg" href="/integrations/vscode">
    Select Ollama models from the Copilot Chat model picker in VS Code.
  </Card>
</CardGroup>

> See the sidebar for all integrations.

Reviewer: ```suggestion
```
